digitalpublicgoods/immunization-calculation-engine.json

```diff
@@ -0,0 +1,88 @@
+{
+  "name": "Immunization Calculation Engine",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://cdsframework.atlassian.net/wiki/spaces/CDSF/pages/14352414/Open-Source+License"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": "N/A"
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://cdsframework.atlassian.net/wiki/spaces/ICE/pages/23494658/Documentation"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "Unknown",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "HIPAA (US)"
+    ],
+    "adherenceSteps": [
+      "https://cdsframework.atlassian.net/wiki/spaces/ICE/pages/23494658/Documentation"
```

Comment:
I would ask this to be a more specific link. This is the same link as the `documentationURL` field above, and I don't see a readily available mention to their Privacy Compliance or Adherence Steps in there.